extensions-core/stats/src/main/java/io/druid/query/aggregation/teststats/ZtestPostAggregator.java

```diff
@@ -131,29 +146,45 @@ private double zScoreTwoSamples(Double s1count, Double p1count, Double s2count,
       convertRate2 = s2count / p2count;
 
       return (convertRate1 - convertRate2) /
-          Math.sqrt((convertRate1 * (1 - convertRate1) / p1count) +
-              (convertRate2 * (1 - convertRate2) / p2count));
-    } catch (Exception ex) {
+             Math.sqrt((convertRate1 * (1 - convertRate1) / p1count) +
+                       (convertRate2 * (1 - convertRate2) / p2count));
+    }
+    catch (IllegalArgumentException ex) {
       return 0;
     }
   }
 
+  public int hashCode()
+  {
+    int result = name != null ? name.hashCode() : 0;
+    result = 31 * result + successCount1.hashCode();
+    result = 31 * result + sample1Size.hashCode();
+    result = 31 * result + successCount2.hashCode();
+    result = 31 * result + sample2Size.hashCode();
+    return result;
+  }
+
   @Override
-  public String toString() {
+  public String toString()
+  {
     return "ZtestPostAggregator{" +
-        "name='"
-        + name
-        + '\''
-        + ", fields="
-        + fields
-        + "}";
+           "name='" + name + '\'' +
+           ", successCount1" + successCount1 +
```

Comment:
Please insert '=' between string and value.

Reply:
will do